docker-compose.deploy.yml

```diff
@@ -36,6 +36,21 @@ services:
     labels:
       - com.centurylinklabs.watchtower.enable=true
 
+  search:
+    image: "getmeili/meilisearch:test-arm2"

```

Comment:
any reason for that specific tag and not just "latest"?

Reply:
It is to be able to run on arm processors, since latest have problems runing on arm. test-arm2 uses the same version as latest. but we could use latest for deployment.